crime-assessment-service/src/main/java/uk/gov/justice/laa/crime/assessmentservice/config/ResourceServerConfiguration.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.justice.laa.crime.assessmentservice.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.Customizer;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.oauth2.server.resource.web.BearerTokenAuthenticationEntryPoint;
+import org.springframework.security.oauth2.server.resource.web.access.BearerTokenAccessDeniedHandler;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.access.AccessDeniedHandler;
+
+@Configuration
+@EnableWebSecurity
+public class ResourceServerConfiguration {
+
+    public static final String SCOPE_CAS_STANDARD = "SCOPE_cas/standard";
+
+    @Bean
+    protected BearerTokenAuthenticationEntryPoint bearerTokenAuthenticationEntryPoint() {
+        BearerTokenAuthenticationEntryPoint bearerTokenAuthenticationEntryPoint =
+                new BearerTokenAuthenticationEntryPoint();
+        bearerTokenAuthenticationEntryPoint.setRealmName("Crime Assessment Service API");
+        return bearerTokenAuthenticationEntryPoint;
+    }
+
+    @Bean
+    public AccessDeniedHandler bearerTokenAccessDeniedHandler() {
+        return new BearerTokenAccessDeniedHandler();
+    }
+
+    @Bean
+    SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        http.sessionManagement(sess -> sess.sessionCreationPolicy(SessionCreationPolicy.STATELESS))
+                .csrf(AbstractHttpConfigurer::disable)
+                .authorizeHttpRequests(auth -> auth.requestMatchers("/open-api/**")
+                        .permitAll()
+                        .requestMatchers("/actuator/**")
+                        .permitAll()
+                        .requestMatchers("/api/**")
+                        .hasAuthority(SCOPE_CAS_STANDARD)
+                        .anyRequest()
+                        .authenticated())
```

Comment:
I suspect this is the doing of the code style, but is it possible to have the `.permitAll()` on the same line as the thing that is being permitted? Makes it so much easier to read and understand.

Reply:
Sad for Spotless, happy for Swagger endpoints 🙂